openapi/components/schemas/ReportJournal.yaml

```
@@ -0,0 +1,27 @@
+type: array
+items:
+  type: object
+  properties:
+    aggregationValue:
+      description: The aggregation field value.
+      type: string
+    bookedMonth:
+      description: Revenue booked month.
+      type: string
+      pattern: '^\d{4}-\d{2}$'
+      example: 2022-02
+    bookedAmount:
+      description: The revenue amount booked for recognition.
```

nitpick: Where possible, we should avoid starting descriptions with "The"

It makes the docs tiresome to read

(I have some really basic writing style guidance for API docs [here](https://product.rebilly.dev/writing-style-guide/api-reference/))

```suggestion
      description: Revenue amount this is booked for recognition.
```

---

```suggestion
      description: Revenue amount which is booked for recognition.
```

---

Oops! :) Sorry